### PR TITLE
kool-untyped 2nd

### PR DESCRIPTION
**Pull request by daejunpark**

### CODE REVIEW

**Comment by daejunpark** [comment]
Jenkins: retest

**Comment by daejunpark** [comment]
Jenkins: test this please

**Comment by daejunpark** [comment]
@grosu please review. 

This PR addresses only parts of your comments that are relatively easy to fix. I want to merge this first, since others will take more time to fix, and I want to make sure future changes will not break this.

My only concern is 8a24fe8143a921108f0b67930d5c97bf3902ce9f, but it is tracked by #1932 and will be fixed correctly later.

File: k-distribution/tutorial/1_k/1_lambda/lesson_9/tests-kore/closed-variable-capture.lambda.out [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Solution: 0
 V
 ---> 
-lambda y . ( ( lambda x . lambda y . ( x y ) ) y )
+lambda y . ` ` lambda x . lambda y . ` x y ` ` y `
```
Comment (grosu) [inline k-distribution/tutorial/1_k/1_lambda/lesson_9/tests-kore/closed-variable-capture.lambda.out:4]: Why are we getting backticks here?

File: k-distribution/tutorial/2_languages/2_kool/1_untyped/kool-untyped.k [hunk 2]
```diff
@@ -210,7 +210,7 @@ for the KOOL object-oriented constructs.
                > Exp "=" Exp             [strict(2), right]
 
   // TODO(KORE): drop klabel once issues #1913 are fixed
-  syntax Ids  ::= List{Id,","}           [klabel("'_,_::Ids")]
+  syntax Ids  ::= List{Id,","}           [klabel("_,_::Ids")]
```
Comment (grosu) [inline k-distribution/tutorial/2_languages/2_kool/1_untyped/kool-untyped.k]: according to @radumereuta , we should not need double quotes int he KLabel at all, so we could just write `klabel(_,_::Ids)`.

File: k-distribution/tutorial/1_k/1_lambda/lesson_9/tests-kore/closed-variable-capture.lambda.out (hunk 1 above)
Comment (daejunpark) [inline k-distribution/tutorial/1_k/1_lambda/lesson_9/tests-kore/closed-variable-capture.lambda.out:4]: It is a bug of unparsing. See #1932. This is a temporary change to ensure that all the other tests pass.

File: k-distribution/tutorial/2_languages/2_kool/1_untyped/kool-untyped.k [hunk 3]
```diff
@@ -565,8 +561,10 @@ from SIMPLE unchanged. */
   rule <k> lookup(L) => V ...</k> <store>... L |-> V:Val ...</store>  [lookup]
 
   syntax KItem ::= env(Map)
-  rule <k> env(Env) => .K ...</k>  <env> _ => Env </env>  [structural]
-  rule `env(_) => .K` ~> env(_)  [structural]
+  rule <k> env(Env) => . ...</k>  <env> _ => Env </env>  [structural]
+  rule `env(_) => .` ~> env(_)  [structural]
```
Comment (grosu) [inline k-distribution/tutorial/2_languages/2_kool/1_untyped/kool-untyped.k:565]: can't we use parentheses here, too?

**Comment by grosu** [comment]
looks very nice, i love it (except for the minor issues I raised)!  thanks, @daejunpark.

File: k-distribution/tutorial/2_languages/2_kool/1_untyped/kool-untyped.k (hunk 3 above)
Comment (daejunpark) [inline k-distribution/tutorial/2_languages/2_kool/1_untyped/kool-untyped.k:565]: not until #1929 is fixed.

Comment (grosu) [inline k-distribution/tutorial/2_languages/2_kool/1_untyped/kool-untyped.k:565]: got it.  please add a todo as well there then, to make sure we don't forget to change it, although the parser will tell us anyway.  i'm ok with it, whichever way you go.

**Pull request merged**